src/java.base/share/classes/java/lang/reflect/AccessFlag.java

```diff
@@ -667,8 +665,6 @@ public enum Location {
         private static final Set<Location> SET_CLASS = Set.of(CLASS);
         private static final Set<Location> SET_CLASS_INNER_CLASS =
             Set.of(CLASS, INNER_CLASS);
-        private static final Set<Location> SET_CLASS_SUPER_VALHALLA =
-                ValhallaFeatures.isEnabled() ? EMPTY_SET : SET_CLASS_INNER_CLASS;
         private static final Set<Location> SET_CLASS_IDENTITY_VALHALLA =
                 ValhallaFeatures.isEnabled() ? SET_CLASS_INNER_CLASS : EMPTY_SET;
```

Comment:
Should this static final field be removed?

Reply:
done